CHANGELOG.md

```diff
@@ -3,6 +3,116 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0-beta.0
+
+_Sep 16, 2021_
+
+🎉 This is the first release of the data grid component with full support of the new MUI 5.0 release 🎉!
+
+A big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- ⚡ Migrate to the new `@mui/material` and `@mui/styles` packages and drop support for `@material-ui/core` (#2515, #2571) @m4theushw
+- 📚 Migrate to the new documentation infrastructure and design (#2441) @DanailH
+- 🚀 Introduce the row menu feature  (#2385) @m4theushw
```

Comment:
Should we add a demo or link to the docs?

Reply:
Totally yes
I just struggle to access the doc and get the correct URL https://next.material-ui.com/components/data-grid/columns/
Probably due to the migration to `mui.com`

Reply:
The docs for the development branch is now at https://next--material-ui-x.netlify.app/components/data-grid/columns/#column-types

Reply:
So in the changelog I add https://mui.com/components/data-grid/columns/#column-types right ?

Reply:
I think so yes